### PR TITLE
ci-operator/config/openshift/cincinnati: Drop all builder caching

### PR DESCRIPTION
**Pull request by vrutkovs**
Supercedes #14518

### CODE REVIEW

File: ci-operator/config/openshift/cincinnati/openshift-cincinnati-master.yaml [hunk 1]
```diff
@@ -32,17 +32,32 @@ images:
...
+raw_steps:
+- pipeline_image_cache_step:
+    commands: |
+      cat <<DOCKERFILE > /opt/Dockerfile
+      FROM centos
+      WORKDIR /go/src/github.com/openshift/cincinnati/
+      COPY . .
+      DOCKERFILE
+    from: centos
+    to: src-dockerfile
```
Comment (LalatenduMohanty) [inline ci-operator/config/openshift/cincinnati/openshift-cincinnati-master.yaml:52]: I wonder if we create the Dockerfile here or we should add the source docker file to the Cincinnati repository.

Comment (vrutkovs) [reply ci-operator/config/openshift/cincinnati/openshift-cincinnati-master.yaml:52]: The safest way is to go with `src-dockerfile`, update `Dockerfile.build` in Cincinnati later and remove `src-dockerfile`. This way we'll know if things broke down

**Review by wking — APPROVED** [review]
Holding in case we want to wait for (some subset of) rehearsals.

/lgtm
/hold

**Comment by vrutkovs** [comment]
/hold cancel

images and e2e need fixes in Cincinnati code

**Pull request merged**